### DISCUSSION TITLE
how to change secret fetching interval?

### DISCUSSION THREAD

**Discussion by vijaySamanuri** [strimzi/kafka-kubernetes-config-provider#26] [Q&A]
i see secrets are fetched every 10 mins interval is there a way to configure this value ?

**Comment by scholzj** [comment]
There is no interval like that. The provider is used inside Kafka configurations and the secrets are fetched when Kafka calls it.

**Reply by vijaySamanuri** [reply]
Thanks for the reply @scholzj 

yes noticed `refresh.topics.interval.seconds` is also used for reevaluating connectorconfig.

i have a follow up question:

i have used following ssl settings in `MirrorSourceConnector` config

```
source.cluster.ssl.truststore.certificates: ${secrets:test/truststore:ca.crt}
source.cluster.ssl.truststore.type: PEM

```

the above settings fetching trust certificates only once but not getting refetched on interval. 
how to support dynamic client ssl config?

**Reply by scholzj** [reply]
I'm not sure what you mean by dynamic config. If you want to change the configuration of the connector, you have to use the REST API or the Strimzi KafkaConnector resources if you use Strimzi. But I do not think Kafka has any support to reload the values from the config providers just because the value changed somewhere.
